packages/web/src/client/components/Cards/CardHome.tsx

```diff
@@ -51,7 +51,7 @@ const CardHomeInner = ({
       <CardText theme={theme}>
         <div>
           {actualTitle ? (
-            <Heading tag="h2" fontStyle="L" weight="$bold">
+            <Heading tag="h2" fontStyle="S" weight="$bold">
```

Comment:
This makes it too small on mobile.

Reply:
@joshuaellis but it's the same 17px in figma

Reply:
shall I make it 20px (slightly bigger)?

Reply:
so I can achieve this without passing `css={{}}` into the component, just using the `fontStyle` prop only?

Reply:
it cannot be 17px in both mobile and desktop, correct? So, if I want it 17 on mobile, it will be 20px on desktop based on how the `getFontStyles` work

Reply:
I am just confused, because Myles said it should be 17px in his screenshot

Reply:
There are lots of ways to make this work, I would focus on making the cleanest solution that is easy to build on if we require too.